### PR TITLE
Fix Issue 68

### PR DESCRIPTION
**Pull request by colin-stubbs**
Suggested way in which to fix issue #68

### CODE REVIEW

File: postfix/os_familymap.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+RedHat:
```
Comment (aboe76) [inline postfix/os_familymap.yaml]: Debian should be moved as wel it could also be used as os_family

**Comment by javierbertoli** [comment]
I think that all `osmap.yml` is more suitable for a `osfamilymap.yml` and probably the wrong grain was used?

**Comment by EvaSDK** [comment]
Agreed it is more likely. osfamilymap can be kept empty or removed until the need arise.

**Comment by aboe76** [comment]
@javierbertoli and @EvaSDK could you please be more specific ?

**Comment by javierbertoli** [comment]
> @javierbertoli and @EvaSDK could you please be more specific?

I'll try :smile: 

It seems to me that all the values currently being set in the `osmap.yml` belong in `osfamilymap.yml`, So we'd probably move them all to `osfamilymap.yml` (you already suggested moving Debian's), and, for now, discard `osmap.yml` or leave it empty.

I *assume* @EvaSDK meant the same, but expressed it the other way round?

**Comment by EvaSDK** [comment]
Indeed, I swapped osmap with osfamilymap. Sorry for the confusion.

**Comment by aboe76** [comment]
@colin-stubbs , can you take the suggestions from @javierbertoli andd @EvaSDK and modify the PR

**Comment by colin-stubbs** [comment]
Shall do

**Comment by javierbertoli** [comment]
@colin-stubbs any chance you can submit the changes we discussed so we can merge this PR?

**Comment by colin-stubbs** [comment]
Sorry guys, I no longer have the systems or the time to go about testing the changes you're proposing.

The pull's here as is, it worked, I'm going to have to leave it as is. Your choice what to do with it.

**Pull request closed without merging**